sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -439,3 +439,117 @@ ALTER TABLE `onsite_mail` CHANGE `owner` `owner` varchar(128) DEFAULT NULL;
 #IfNotColumnType openemr_postcalendar_events pc_facility int(11)
 ALTER TABLE `openemr_postcalendar_events` CHANGE `pc_facility` `pc_facility` int(11) NOT NULL DEFAULT '0' COMMENT 'facility id for this event';
 #Endif
+
+#IfNotTable medex_icons
+CREATE TABLE IF NOT EXISTS `medex_icons` (
+  `i_UID` int(11) NOT NULL AUTO_INCREMENT,
+  `msg_type` varchar(50) NOT NULL,
+  `msg_status` varchar(10) NOT NULL,
+  `i_description` varchar(255) NOT NULL,
+  `i_html` text NOT NULL,
```

Comment:
Need to separate out the html code from the labels here(ie. can build the html from information in the tables when pull this). This is a security issue, since you will not be able to html escape these entries(so, if a bad actor was able to place html code in i_html, they could do cross scripting easily and even possibly run php code on the server). Dealing with this would then also make it much easier(ie. not impossible :) ) to support translation.